### DISCUSSION TITLE
Amiberry ARM (RetroPie) and Amiga Forever 8 Kickstart ROMS

### DISCUSSION THREAD

**Discussion by Randune** [BlitterStudio/amiberry#910] [Q&A] [answered]
Hi all, I'm trying to use the Amiga Forever 8 Premium Edition Kickstart roms (they come with the accompanying rom.key file) with Amiberry on my RetroPie.  I put the Kickstart roms in the /home/pi/RetroPie/BIOS folder along with the rom.key file but upon loading for instance a .lha whdload game it it just goes to a black screen.  Upon checking the /dev/shm/runcommand.log file it says that it cannot find the system rom.  I then tried with another rom procured online and it works fine.  So I'm just trying to figure out if Amiberry is capable of using the Amiga Forever Kickstart roms and if so, what do I need to do, is there any special configuration required?

Any help would be appreciated, thanks in advance!

**Comment by midwan** [comment]
Yes, Amiga Forever ROMs should work just fine. In fact, those are what I also have and use.
Have you launched AF once on the PC, before copying the ROMs over? In newer versions it decrypts them the first time it starts up, not sure when that started happening. 
And did you do a Rescan ROMs on Amiberry's side after you have copied them? You can find that button on the Paths panel.

**Comment by Randune** [comment, marked as answer]
I figured out the issue, what I was doing was copying the kickstart roms directly off the Amiga Forever DVDs along with the rom.key file then putting those in the /home/pi/RetroPie/BIOS folder but amiberry doesn't seem to use the rom.key file to decrypt the kickstart roms.  So what I found out like you say @midwan is install Amiga Forever on my windows VM which decrypts the kickstarts then copy those kickstart roms to my /home/pi/RetroPie/BIOS folder and did a ROM rescan in amiberry and voila! It works.  Thank you for your assistance.
